src/index.test.js

```diff
@@ -67,6 +68,21 @@ describe('hotfix cmp.init', () => {
 		expect(window.guCmpHotFix.initialised).toBe(true);
 	});
 
+	it('warn if two versions are running simultaneously', () => {
+		const spy = jest.spyOn(global.console, 'warn');
+		cmp.init({ isInUsa: false });
+		const cmpVersions = [window.guCmpHotFix.cmp.version, 'mockedVersion'];
+		// eslint-disable-next-line prefer-destructuring
+		window.guCmpHotFix.cmp.version = cmpVersions[1];
```

Comment:
I wasn't quite clear what was going on here. does this capture it?

```suggestion

		const currentVersion = window.guCmpHotFix.cmp.version;
		window.guCmpHotFix.cmp.version = 'mockVersion';
```

Reply:
I was wondering which way was clearer. More explicit is better!